content/plugins/aws.md

```diff
@@ -19,22 +22,26 @@ For more information about the library, please refer to: [http://boto.readthedoc
 * Python Versions:
   * 2.7.x
 * An account with AWS
-  * Your AWS [Access Keys](http://docs.aws.amazon.com/AWSSecurityCredentials/1.0/AboutAWSCredentials.html#)
 
 
 # Compatibility
 
-The AWS EC2 plugin currently only supports classic EC2.
-
 The AWS plugin uses the following client libraries:
 
-  * [Boto client](https://github.com/boto/boto) - 2.34.0
+  * [Boto client](https://github.com/boto/boto) - 2.38.0
 
 {{% gsNote title="Note" %}}
 This version of Boto EC2 Connection supports (AWS) APIVersion = '2014-10-01'.
 {{% /gsNote %}}
 
 
+# Some terminology
+
+* VPC is a virtual private cloud, which is AWS's a virtual private network product.
+* EC2 Classic is a public cloud. Often you might see EC2 Classic VPC, this refers to the network that EC2 Classic resources are provisioned on.
```

Comment:
VPC is public cloud as well.

EC2-Classic is the original release of Amazon EC2. With this platform, instances run in a single, flat network that is shared with other customers.